src/routes/balances/handlers_v2.rs

```diff
@@ -52,7 +55,7 @@ pub async fn balances(
     let token_prices: Vec<TokenPrice> =
         get_token_prices(context, &info_provider, &backend_balances).await;
 
-    let service_balances: Vec<Balance> = backend_balances
+    let mut service_balances: Vec<Balance> = backend_balances
```

Comment:
Is it necessary for this to be mutable? I would expect `total_fiat` requiring to be `mut`

Reply:
It is required because of the sorting of the vector.